mountpoint-s3/src/fstab.rs

```diff
@@ -0,0 +1,227 @@
+use anyhow::anyhow;
+use clap::Parser;
+use mountpoint_s3_fs::cli::CliArgs;
+use std::env;
+use std::ffi::OsString;
+
+#[derive(Parser, Debug)]
+pub struct FsTabCliArgs {
+    pub bucket_name: String,
+    #[clap(value_name = "DIRECTORY")]
+    pub mount_point: String,
+    #[clap(short = 'o')]
+    pub options: String,
```

Comment:
Should we make this a `FsTabOptions` and use a [value parser](https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_parser)?

Reply:
In the most recent commit, I removed `FsTabOptions` and opted to go with a simpler approach of pre&post processing of the CLI arguments prior to parsing them as `CliArgs` .

Reply:
I could still feasibly use a value parser to split by commas and do all the pre-processing however if we think that's a good idae